freeipa/src/main/java/com/sequenceiq/freeipa/service/image/ImageService.java

```diff
@@ -29,6 +36,8 @@
 
     private static final String DEFAULT_REGION = "default";
 
+    private static final Pattern VERSION_PATTERN = Pattern.compile("^([0-9]+\\.[0-9]+\\.[0-9]+-([d,r][e,c][v]?\\.|[b]))[0-9]+$");
```

Comment:
so I think this will match something 2.2.2-dcv, 2.2.2-rev etc. I think we should match only `dev` or `rc`